tee-worker/litentry/pallets/identity-management/src/lib.rs

```diff
@@ -196,6 +198,21 @@ pub mod pallet {
 			if let Some(c) = IDGraphs::<T>::get(&who, &identity) {
 				ensure!(!c.is_verified, Error::<T>::IdentityAlreadyVerified);
 			}
+			if let Identity::Substrate { network, address } = identity {
+				// see all the address prefix:
+				// https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json
+				let ss58_prefix = T::SS58Prefix::get();
+				if (network == SubstrateNetwork::Litentry && ss58_prefix == 31)
+					|| (network == SubstrateNetwork::Litmus && ss58_prefix == 131)
```

Comment:
This might not work though. `T` here is the sgx-runtime, but we need the information from the parent chain (`SS58Prefix` for sgx-runtime is 42 btw - standard substrate).

The first idea that comes to my mind is to change `user` in `create_identity` to Identity type and we only need to verify:
- the user's network type matches the parachain (in teerex)
- the linked identity is different from the user's identity (in tee-worker)

Ideally we need ts-test for it